helix-term/src/ui/statusline.rs

```diff
@@ -354,16 +355,21 @@ where
             .as_ref()
             .map(|p| p.to_string_lossy())
             .unwrap_or_else(|| SCRATCH_BUFFER_NAME.into());
-        format!(
-            " {}{} ",
-            path,
-            if context.doc.is_modified() { "[+]" } else { "" }
-        )
+        format!(" {} ", path)
     };
 
     write(context, title, None);
 }
 
+fn render_file_modified<F>(context: &mut RenderContext, write: F)
+where
+    F: Fn(&mut RenderContext, String, Option<Style>) + Copy,
+{
+    let title = format!(" {} ", if context.doc.is_modified() { "[+]" } else { "" });
```

Comment:
The surrounding space ends up putting this element pretty far apart from the file name. I think it's ok to only write `"[+]"` here without the surrounding space given the element's intended usage

Reply:
I think the `else` branch should be three spaces: that would allow this element to be used before the FileName element without causing a jump when the file becomes modified

Reply:
Changed